codes/classical/bits/repetition code.yml

```diff
@@ -0,0 +1,58 @@
+#######################################################
```

Comment:
Please integrate into existing repetition.yml file (the file name for your file is different, so Github did not pick it up).